sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -63,9 +69,6 @@ export enum RecordTypeCodes {
   ECertPartTimeHeader = "01",
   ECertPartTimeRecord = "02",
   ECertPartTimeFooter = "99",
-  ECertPartTimeFeedbackHeader = "001",
```

Comment:
I guess `RecordTypeCodes` is used while creation and response reading right? If so, i think we need to adjust the comment line 57

Reply:
Comment adjusted.